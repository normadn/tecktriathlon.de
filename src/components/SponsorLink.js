import React from "react";
import { OutboundLink } from "gatsby-plugin-google-analytics";

export default (props) => (
  <OutboundLink
    href={props.url}
    target="_blank"
    rel="external noopener"
    data-cy="sponsor-link"
  >
    {props.children}
  </OutboundLink>
);
