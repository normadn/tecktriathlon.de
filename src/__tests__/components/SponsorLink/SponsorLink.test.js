import React from "react";
import SponsorLink from "../../../components/SponsorLink";
import { shallow } from "enzyme";

describe("SponsorLink", () => {
  it("renders correctly", () => {
    expect(
      shallow(<SponsorLink url="https://www.youtube.com/watch?v=6n3pFFPSlW4" />)
    ).toMatchSnapshot();
  });
});
