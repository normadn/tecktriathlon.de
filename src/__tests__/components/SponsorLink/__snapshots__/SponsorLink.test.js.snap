// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SponsorLink renders correctly 1`] = `
<OutboundLink
  data-cy="sponsor-link"
  href="https://www.youtube.com/watch?v=6n3pFFPSlW4"
  rel="external noopener"
  target="_blank"
/>
`;
